catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/FileSystemPersistenceProvider.java

```diff
@@ -53,50 +56,34 @@
 
   private static final String SER_REGEX = "\\.ser";
 
-  private String mapName = "default";
+  @VisibleForTesting static final String STORAGE_ROOT_PROPERTY = "storage";
 
-  FileSystemPersistenceProvider(String mapName) {
+  private Properties properties;
+
+  private Path defaultPersistencePath;
+
+  private Path persistencePath;
+
+  FileSystemPersistenceProvider(String mapName, Properties properties) {
```

Comment:
Same as above, think it probably makes sense change this to just take the path to the storage directory instead of the properties object